### PR TITLE
Add Contact Form

### PR DESCRIPTION
**Pull request by SaboorSohaib**
### Requirement

-the contact form in the footer.

**Please check my project if there is any problem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
...
-        
-        
+        <section class="contact-form">
+            <h2 class="contact-box contact-text">
+                I'm always interested in hearing about new projects,
+                 so if you'd like to chat please get in touch.
+            </h2>
+            <form action="submit" class="contact-form">
+                <div class="name">
+                    <input type="text" maxlength="30"  placeholder="Full name" required>
```
Comment (codepantha) [inline index.html]: - [ ] Your contact form is currently not submitting successfully. Kindly use the `action` and `method` attributes to integrate with the [formspree service](https://formspree.io/html/)
**NB: ** 
**action value should be similar to https://formspree.io/f/{form_id}**
**method should equal "POST"**

File: index.html [hunk 2]
```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
...
+                </div>
+                <div class="email">
+                    <input type="email" name="email"  placeholder="Email address" required>
+                </div>
+                <div >
+                    <textarea class="message" name="textarea"  maxlength="500" required placeholder=" Enter text here"></textarea>
+                </div>
+                <button type="submit" class="contact-btn contact-btn-text ">Get in touch</button>
+                <div class="fotter-icon">
```
Comment (codepantha) [inline index.html:211]: - [ ] Kindly implement the `:hover` and pressed states of your button as specified in the Figma Design shown below 👇 
image

File: index.html [hunk 3]
```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
...
+            </h2>
+            <form action="submit" class="contact-form">
+                <div class="name">
+                    <input type="text" maxlength="30"  placeholder="Full name" required>
+                </div>
+                <div class="email">
+                    <input type="email" name="email"  placeholder="Email address" required>
+                </div>
+                <div >
+                    <textarea class="message" name="textarea"  maxlength="500" required placeholder=" Enter text here"></textarea>
```
Comment (codepantha) [inline index.html:208]: - [ ] The space between each input element in your form is wider than the Figma design. Kindly adjust your design accordingly 🙏 
### Figma Design
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @SaboorSohaib 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Used GitHub Flow
✔️ Good commit history
✔️ Good UI design
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] Since it's a collaborative project, kindly include your coding partner's details in the "Authors" section of your README

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
...
+                </div>
+                <button type="submit" class="contact-btn contact-btn-text ">Get in touch</button>
```
Comment (Nemwel-Boniface) [inline index.html:210]: - [ ] As requested by the previous reviewer, kindly implement button states. These include on hover, on click, active etc. If you may not be sure of how to implement this, have a look at [this awesome resource](https://zellwk.com/blog/style-hover-focus-active-states/).
- [ ] It also seems like your button text is not fitting correctly in the button width. Kindly have a look at it.

**This is what I mean:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @SaboorSohaib           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**